src/components/Hero/Hero.tsx

```diff
@@ -59,14 +55,15 @@ export interface HeroProps {
   heroType?: HeroTypes;
   /** ID that other components can cross reference for accessibility purposes. */
   id?: string;
-  /** Object used to create and render the `Image` component. Note that only
-   * `alt`, `component`, and `src` are the available attributes to pass. If
-   * `imageProps.alt` is left blank, a warning will be logged to the console and
+  /** Object used to create and render the `Image` component. You can pass `component`
+   * (with its own internal props, which will override) or `src`, `alt`, `id`, `fallBackSrc`, and `onError`.
+   * If `imageProps.alt` is left blank, a warning will be logged to the console and
    * will cause accessibility issues. For `imageProps.src`, it will only work for
-   * the "secondary", "fiftyFifty" and "campaign" `Hero` types; Note: `imageProps.src`
-   * can only be used in conjunction with `backgroundImageSrc` for the "campaign"
-   * `Hero` type. Note: not all `Hero` variations utilize this prop. */
-  imageProps?: HeroImageProps;
+   * the "secondary", "fiftyFifty" and "campaign" `Hero` types. */
+  imageProps?: Pick<
```

Comment:
This is fine but any reason why it's directly here rather than in the interface?

Reply:
This is how it renders in storybook which is not better or worse, just different.

<img width="712" alt="Screenshot 2024-10-15 at 6 16 54 PM" src="https://github.com/user-attachments/assets/a9e8a33b-64cb-488e-be1c-845b0e5e7ac2">